docs/source/installation/linux.rst

```diff
@@ -17,6 +17,17 @@ Install manim via pypi::
     # pip3 install manimlib
 
 OR Install manim via the git repository with venv::
+NOTE : Make sure that the python version you are using is python3.7 because manim works only for that version . If not, you will get errors when running the old_projects.If the python3 version is not 3.7 then upgrade your version usimg the following steps:
+     
+sudo apt-get install -y software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt-get update
+sudo apt-get install build-essential libpq-dev libssl-dev openssl libffi-dev zlib1g-dev
```

Comment:
Do you know why openssl/libssl is needed?

Reply:
PIP uses HTTPS connection to Python package index by default so that may be the reason.